### ISSUE TITLE
Version 1.0.4 not compatible with latest Coverlet release (2.1).

### ISSUE THREAD

**Issue by NeilCross** [Romanx/Cake.Coverlet#1]
This project has been broken by the latest changes in Coverlet.

Build task property names have changed for output, please see [https://github.com/tonerdo/coverlet/blob/master/src/coverlet.msbuild/coverlet.msbuild.props](url).

Changes are: 
- It appears that CoverletOutputDirectory and CoverletOutputName has been merged into CoverletOutput
- New properties ThresholdType and ExcludeByFile

**Comment by Romanx** [comment]
Hi there,

Thanks for reporting this. I'll fix it and get a release out tonight.

Thanks,

**Comment by Romanx** [comment]
Hi there,

Just looked at this now and it seems that the fix I wanted to add isn't entirely supported on Coverlet's side. I've opened and issue there which can be found [here.](https://github.com/tonerdo/coverlet/issues/146).

For some feedback would it work for you if you had to provide an extension to the CoverletOutputName until they fix their end? I'm not sure if it's a good enough workaround to release 1.1 now.

**Comment by Romanx** [comment]
@NeilCross Coverlet has just had some code merged that fixes my issue and we're just waiting on the next release which should be soon. Thanks for the wait 👍

**Comment by Romanx** [comment]
@NeilCross this has just been released to nuget. Pleae let me know if you have any issues.
